modules/apps/object/object-admin-rest-test/src/testIntegration/java/com/liferay/object/admin/rest/resource/v1_0/test/ObjectDefinitionResourceTest.java

```diff
@@ -558,6 +568,61 @@ public void testPostObjectDefinition() throws Exception {
 		assertEquals(postObjectDefinition, randomObjectDefinition);
 		assertValid(postObjectDefinition);
 
+		// Permissions
+
```

Comment:
@vngoliveira Aparantely, it's a pattern for all test commentary blocks initialize it's own object definition. So please do that:

`randomObjectDefinition = randomObjectDefinition();`

and adapt your logic to it, avoiding reusing from another test.